src/components/ExternalProject.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Link = styled.a`
+  display: block;
+  border: 0;
+  margin: 0 20px;
+  text-decoration: none;
+  transition: opacity 0.1s ease-in;
+
+  &:hover {
+    opacity: 0.6;
+  }
+
+  @media (max-width: 980px) {
+    & {
+      width: 100%;
+      margin-right: 0;
+      margin-bottom: 30px;
+      margin-left: 0;
+    }
+  }
+`;
+
+const ImageWrapper = styled.div`
+  text-align: center;
+`;
+
+const Image = styled.img`
+  height: 50px;
+
+  @media (max-width: 980px) {
+    & {
+      height: 100px;
+    }
+  }
+`;
+
+export default ({url, logo, name}) => (
+  <Link href={url} target="_blank">
```

Comment:
You also need `rel="noopener"` with `target=_blank` to prevent a vulnerability.

Reply:
Where was the warning coming from? A Gatsby-specific `eslint` plugin?

Reply:
```
Module Warning (from ./node_modules/eslint-loader/index.js): 
```

I’m clueless on who calls eslint because I don’t have a config or package `eslint-*` in my project

Reply:
The rule is `react/jsx-no-target-blank` and it’s added from the `react-app` configuration preset:

https://github.com/facebook/create-react-app/blob/master/packages/eslint-config-react-app/index.js#L234

It’s supposed to support custom link elements by adding them in the eslint’s `settings` object like so:

```json
"settings": {
  "linkcomponents": [
    "link"
  ]
}
```

But it still doesn’t raise anything 🤷‍♂️ I’ll investigate later!